File: pyhton64/nivel4/exer36.py
qtd = 10
valor = []
for i in range(qtd):
    valor.append(int(input('Valores: ')))

media = sum(valor) / len(valor)
print(f'Maior: {max(valor)}\nMenor: {min(valor)}\nMédia: {media}')

File: java64/nivel4/exer36.java
import java.util.Arrays;
import javax.swing.JOptionPane;

public class exer36 {
    public static void main(String[] args) {
        int quantidade = 10, media, total = 0;
        int[] valores = new int[quantidade];

        for (int i = 0; i < valores.length; i++) {
            valores[i] = Integer.parseInt(JOptionPane.showInputDialog("" + (i + 1) + "° valor: "));
            total += valores[i];
        }

        media = total / quantidade;
        Arrays.sort(valores);
        JOptionPane.showMessageDialog(null, "Maior: " + valores[quantidade - 1] + "\nMenor: " + valores[0] + "\nMédia: " + media);
    }
}